ledger-service/http-json/src/main/scala/com/digitalasset/http/Main.scala

```diff
@@ -123,6 +123,18 @@ object Main {
             terminate()
             System.exit(ErrorCodes.StartupError)
         }
+      case (Some(dao), _) =>
+        Try(dao.isValid(120).unsafeRunSync()).toEither match {
+          case Right(false) =>
+            logger.error("Database connection is not valid.")
+            terminate()
+            System.exit(ErrorCodes.StartupError)
+          case Left(e) =>
+            logger.error("Failed to connect to the database", e)
+            terminate()
+            System.exit(ErrorCodes.StartupError)
+          case _ =>
```

Comment:
```suggestion
          case Right(true) =>
```
not a huge fan of wildcard matches when they’re easily avoidable. They tend to bite you when they suddenly end up catching more than you wanted to after refactoring.